### PR TITLE
docs(contributing): update contribution process

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Contributing
 
+We'd love for you to contribute to our source code and to make **Webex SDK Component Adapter** even better than it is today!
+If you would like to contribute to this repository by adding features, enhancements or bug fixes, you must follow our process:
+
+  1. [Create an issue](https://github.com/webex/sdk-component-adapter/issues) to propose your solution _before_ you get coding
+  2. Let core members know about your proposal by posting a message in the [contributor's Webex Teams space](https://eurl.io/#Bk9WGfRcB)
+  3. A core member will review your proposal and if necessary may suggest to have a meeting to better understand your approach
```
Comment (lalli-flores) [inline CONTRIBUTING.md]:
```suggestion
  3. A core member will review your proposal and if necessary may suggest to have a meeting to better understand your approach
      3a. You are welcomed you join our [weekly review meeting](https://cisco.webex.com/m/f4ebbec6-c306-49ca-83f4-fb2d098fc946) (Thursdays, 11:30a-12:30p PST) to propose your contribution as well
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+  4. If your proposal is approved you should start coding at this point
+  5. We recommend opening a draft PR to receive feedback before finalizing your solution
```
Comment (lalli-flores) [inline CONTRIBUTING.md]:
```suggestion
  5. We recommend opening a draft PR to receive feedback before finalizing your solution
      5a. When opening a draft PR, specify with PR comments where in the code you would like to get feedback
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+  6. Before opening a PR ensure **all** [PR guidelines](#pull-request-guidelines) are followed
+  7. Let core members know about your PR by posting a message in the [contributor's Webex Teams space](https://eurl.io/#Bk9WGfRcB)
+  8. The PR will be reviewed by core members and leave feedback, if any
```
Comment (lalli-flores) [inline CONTRIBUTING.md]:
```suggestion
  8. The PR will be reviewed by core members and leave feedback, if any
      8a. If a PR is too large, you may be asked to break it down into multiple smaller-scoped PRs
```

**Comment by adamweeks** [comment]
@lalli-flores suggestions committed.

**Pull request merged**